source/game/field/obj/ObjectTwistedWay.cc

```diff
@@ -0,0 +1,355 @@
+
+/// @addr{0x80813c40}
+void ObjectTwistedWay::init() {}
+
+/// @addr{0x80813cfc}
+void ObjectTwistedWay::calc() {
+    if (!System::RaceManager::Instance()->isStageReached(System::RaceManager::Stage::Race)) {
+        m_ac++;
+    }
+}
+
+/// @addr{0x80814910}
+u32 ObjectTwistedWay::loadFlags() const {
+    return 1;
+}
+
+/// @addr{0x8081490c}
+void ObjectTwistedWay::createCollision() {}
+
+/// @addr{0x80814908}
+void ObjectTwistedWay::calcCollisionTransform() {}
+
+/// @addr{0x808148f8}
+f32 ObjectTwistedWay::getCollisionRadius() const {
+    return COLLISION_RADIUS;
+}
+
+/// @addr{0x808148b8}
+bool ObjectTwistedWay::checkPointPartial(const EGG::Vector3f &v0, const EGG::Vector3f &v1,
+        KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut) {
+    return checkSpherePartialImpl(0.0f, v0, v1, flags, pInfo, pFlagsOut, 0);
+}
+
+/// @addr{0x808148c8}
+bool ObjectTwistedWay::checkPointPartialPush(const EGG::Vector3f &v0, const EGG::Vector3f &v1,
+        KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut) {
+    return checkSpherePartialPushImpl(0.0f, v0, v1, flags, pInfo, pFlagsOut, 0);
+}
+
+/// @addr{0x808148d8}
+bool ObjectTwistedWay::checkPointFull(const EGG::Vector3f &v0, const EGG::Vector3f &v1,
+        KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut) {
+    return checkSphereFullImpl(0.0f, v0, v1, flags, pInfo, pFlagsOut, 0);
+}
+
+/// @addr{0x808148e8}
+bool ObjectTwistedWay::checkPointFullPush(const EGG::Vector3f &v0, const EGG::Vector3f &v1,
+        KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut) {
+    return checkSphereFullPushImpl(0.0f, v0, v1, flags, pInfo, pFlagsOut, 0);
+}
+
+/// @addr{0x80814958}
+bool ObjectTwistedWay::checkSpherePartial(f32 radius, const EGG::Vector3f &v0,
+        const EGG::Vector3f &v1, KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut,
+        u32 timeOffset) {
+    return checkSpherePartialImpl(radius, v0, v1, flags, pInfo, pFlagsOut, timeOffset);
+}
+
+/// @addr{0x80814ea8}
+bool ObjectTwistedWay::checkSpherePartialPush(f32 radius, const EGG::Vector3f &v0,
+        const EGG::Vector3f &v1, KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut,
+        u32 timeOffset) {
+    return checkSpherePartialPushImpl(radius, v0, v1, flags, pInfo, pFlagsOut, timeOffset);
+}
+
+/// @addr{0x80815444}
+bool ObjectTwistedWay::checkSphereFull(f32 radius, const EGG::Vector3f &v0, const EGG::Vector3f &v1,
+        KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut, u32 timeOffset) {
+    return checkSphereFullImpl(radius, v0, v1, flags, pInfo, pFlagsOut, timeOffset);
+}
+
+/// @addr{0x80815d64}
+bool ObjectTwistedWay::checkSphereFullPush(f32 radius, const EGG::Vector3f &v0,
+        const EGG::Vector3f &v1, KCLTypeMask flags, CollisionInfo *pInfo, KCLTypeMask *pFlagsOut,
+        u32 timeOffset) {
+    return checkSphereFullPushImpl(radius, v0, v1, flags, pInfo, pFlagsOut, timeOffset);
+}
+
+/// @addr{0x807FB54C}
```

Comment:
Wrong address, this is for ObjectAurora. And so are all the addresses below this I think